examples/gio_action_impl/action.rs

```diff
@@ -0,0 +1,77 @@
+use gio::{prelude::*, subclass::prelude::*};
+use glib;
+
+mod imp {
+    use super::*;
+    use std::cell::OnceCell;
+
+    #[derive(glib::Properties, Default)]
+    #[properties(wrapper_type = super::RenamedAction)]
+    pub struct RenamedAction {
```

Comment:
The properties macro allows overriding properties by the way

Reply:
I gave it a try but received an error 

```
(process:718176): GLib-GObject-CRITICAL **: 22:59:07.010: When installing property: type 'ExampleRenamedAction' already has a property named 'name'
```

I guess it's because of the sequence of initialization. When I define a class and implement an interface then class installs properties first and interface fails to install them.

Reply:
The way how you do it now should work or not?

Reply:
It does not work. I guess It works as expected in the cases like 

```
class A implements IFace

class B extends A implements IFace {
   override property x of IFace
}
```

where initialization sequence is: `A`, `IFace`, `B`. So, when `B` is initializing `IFace` is already there and could be overriden.

but does not work for 

```
class B implements IFace {
   override property x of IFace
}
```

where initialization sequence is: `B`, `IFace`. So, `B` has nothing to override yet.